docs/schemas/resource/IfcRepresentationResource/Entities/IfcCoordinateOperation.md

```diff
@@ -1,15 +1,40 @@
+
+An _IfcCoordinateOperation_ allows to connect:
+- an _IfcGeometricRepresentationContext_ &ndash; the base reference system of a virtual model
+- to an _IfcCoordinateReferenceSystem_ &ndash; the coordinate system which is related to the real world by datums.
+
+> EXAMPLE  This can is done by setting:
+> - the _IfcCoordinateOperation.SourceCRS_ to _IfcGeometricRepresentationContext_
+> - the _IfcCoordinateOperation.TargetCRS_ either to:
+>    -	_IfcProjectedCRS_, for defined easting, northing and orthogonal height
+>    -	_IfcGeographicCRS_, for defined latitude, longitude and ellipsoidal height
+
+> NOTE  In IFC, _IfcProjectedCRS_ shall be used to represent a compound CRS. See _IfcProjectedCRS_ for further details.
 
 { .extDef}
 > NOTE  Definition from OpenGIS Abstract Specification, Topic 2:
-> If the relationship between any two coordinate reference systems is known, coordinates can be transformed or converted to another coordinate reference system. Coordinate operations are divided into two subtypes: { .note}
+> If the relationship between any two coordinate reference systems is known, coordinates can be transformed or converted to another coordinate reference system. Coordinate operations are divided into two subtypes:
+>
 > * _Coordinate conversion_ &ndash; mathematical operation on coordinates that does not include any change of datum. The best-known example of a coordinate conversion is a map projection. The parameters describing coordinate conversions are defined rather than empirically derived. Note that some conversions have no parameters.
+>
 > * _Coordinate transformation_ &ndash; mathematical operation on coordinates that usually includes a change of datum. The parameters of a coordinate transformation are empirically derived from data containing the coordinates of a series of points in both coordinate reference systems. This computational process is usually &lsquo;over-determined&rsquo;, allowing derivation of error (or accuracy) estimates for the transformation. Also, the stochastic nature of the parameters may result in multiple (different) versions of the same coordinate transformation. Because of this several transformations may exist for a given pair of coordinate reference systems, differing in their transformation method, parameter values and accuracy characteristics.
 
+There are three types of _IfcCoordinateOperation_. The one to be used depends on the specific use case:
+
+* _IfcMapConversion_ and _IfcMapConversionScaled_ involve a **transformation**. These coordinate operations convey that the local engineering coordinate system of the virtual model (which is a topocentric CRS) is put in place in the referenced _IfcProjectedCRS_. 
+
+* _IfcRigidOperation_ involves a **translation** in the (x,y,z) plane. This coordinate operation convey that the whole virtual model is translated in the same way in (x,y,z).
```

Comment:
```suggestion
* _IfcRigidOperation_ involves a **translation** in the 3D space. This coordinate operation convey that the whole virtual model is translated in the same way. For example, using lengths to translate along x,y,z; or using angles for a 2D translation (e.g., lambda, phi), plus a change in height.
```

Reply:
@pjanck better? Should we repeat/move this to IfcRigidOperation documentation?

Reply:
Better :) Yes, consider shortening this and moving the majority to `IfcRigidOperation` documentation.